benchmark/buffers/buffer_zero.js

```diff
@@ -3,16 +3,18 @@
 const common = require('../common.js');
 
 const bench = common.createBenchmark(main, {
-  n: [1024]
+  n: [1024],
+  type: ['buffer', 'string']
 });
 
-const zero = Buffer.alloc(0);
+const zeroBuffer = Buffer.alloc(0);
+const zeroString = '';
 
 function main(conf) {
   var n = +conf.n;
   bench.start();
   for (let i = 0; i < n * 1024; i++) {
-    Buffer.from(zero);
+    conf.type === 'buffer' ? Buffer.from(zeroBuffer) : Buffer.from(zeroString);
```

Comment:
Sorry if I'm being unclear.  What I mean is that the overhead of the check and the multiplication may affect the benchmark results.  They probably don't but it would be good to check.